pFront/src/app/app.component.ts

```diff
@@ -1,10 +1,121 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { ICity } from '../model/icity';
+import { Router, RouterLink } from '@angular/router';
+import { WeatherComponent } from './weather/weather.component';
+import { OrdersComponent } from './orders/orders.component';
+import { every } from 'rxjs';
+import { IOrders } from '../model/iorders';
+import { IpService } from './ip.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrl: './app.component.scss'
+  styleUrls: ['./app.component.scss'],
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
+
+
   title = 'pFront';
+
+  cities: ICity[] = [];
+
+  orders: IOrders[] = [];
+
+  constructor(private router: Router,private ip : IpService) {}
+
+  ngOnInit(): void {
+    this.getCities().then((cities) => {
+      this.cities = cities;
+      console.log(this.cities); // Log cities to check the structure
+    });
+
+    this.getOrders().then((orders) => {
+      this.orders = orders;
+      console.log(this.orders); // Log cities to check the structure
+    });
+
+  }
+
+  async getCities(): Promise<ICity[]> {
+    return await fetch('http://'+this.ip.Ip+':8080/api/city/getAllCities')
```

Comment:
Proponuję zastosowanie stałych dla ścieżek - umożliwia to łatwiejsze modyfikowanie i dostęp